kart/status.py

```diff
@@ -100,8 +105,33 @@ def status(ctx, output_format):
 
     if output_format == "json":
         dump_json_output({"kart.status/v2": jdict}, sys.stdout)
-    else:
+    else:       
```

Comment:
This code is looking good as far as it goes - my next challenge to you is to update it to be more in line with the existing code, which has the following pattern:

1. Generate the status in a JSON-like-format.
2. a) If "json": dump that status as JSON
2. b) Else if "text": convert the status to text and print it

There's a couple of upsides to doing it this way - anything we expose in "text" output we are forced to also expose in "json" output, and, we can guarantee that the output is consistent (ie, if we output that there are untracked tables when doing "json" output, then we will be sure to also output it when doing "text" output.

I don't know exactly where the untracked tables should go in the JSON output. For now, you could put them in a list under "workingCopy" called "untrackedTables", and hopefully we'll know if that's a good place by by the time this is ready to merge.